docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx

```diff
@@ -213,6 +228,85 @@ To add a Task to the end of a Mission:
 
     :::
 
+### Custom Tasks  {#custom-task}
+
+Users can create their own tasks following a specific template and place them within the **/onav_log/custom_tasks** directory.

```

Comment:
@dobrienCP The user adds custom tasks on the host, not inside the docker, right? Are we sure that "/onav_log/" is the right place to be adding custom tasks? I'm unclear on why customer code is going in a log directory.

Reply:
This is correct the user adds custom tasks on the host in that folder. I can change which folder we are pointing at I picked that one as I thought we had other items going in there as well but you're right it should be straight forward to create a new folder that they can put custom tasks into we'll just need to add it as another volume. I'll change this to ~/custom_tasks.